backend/infrahub/transformations/tasks.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any
+
+from prefect import flow
+
+from infrahub.git.repository import get_initialized_repo
+from infrahub.log import get_logger
+from infrahub.services import services
+
+from .models import TransformJinjaTemplateData, TransformPythonData
+
+log = get_logger()
+
+
+@flow(name="transform-render-python")
+async def transform_python(message: TransformPythonData) -> Any:
+    service = services.service
+
+    repo = await get_initialized_repo(
+        repository_id=message.repository_id,
+        name=message.repository_name,
+        service=service,
+        repository_kind=message.repository_kind,
+    )
+
+    transformed_data = await repo.execute_python_transform(
+        branch_name=message.branch,
+        commit=message.commit,
+        location=message.transform_location,
+        data=message.data,
+        client=service.client,
+    )
+
+    return transformed_data
+
+
+@flow(name="transform-render-jinja2")
+async def transform_render_jinja2_template(message: TransformJinjaTemplateData) -> str:
+    service = services.service
+
+    repo = await get_initialized_repo(
+        repository_id=message.repository_id,
+        name=message.repository_name,
+        service=service,
+        repository_kind=message.repository_kind,
+    )
+
+    rendered_template = await repo.render_jinja2_template(
+        commit=message.commit, location=message.template_location, data={"data": message.data}
+    )
+
+    return rendered_template
```

Comment:
I'm not completely sure about the location of this file. I don't have a better suggestion just now but I think we should consider the overall structure of our modules soon and consider moving this at that point.

Reply:
Agreed